### ISSUE TITLE
\ensuremath does not ensure math

### ISSUE THREAD

**Issue by Lycanic** [chirun-ncl/chirun#212]
\ensuremath should toggle to math mode when used outside of math mode, as in the pdf compilation. Currently, it appears to be ignored when outside of math mode.

Minimal Failing example:
```
\documentclass{article}
\begin{document}

$\ensuremath{\chi}$

\ensuremath{\chi}

\end{document}
```

**Comment by christianp** [comment]
Yep: [the plasTeX implementation of ensuremath notes that it's wrong and needs to be filled in](https://github.com/plastex/plastex/blob/master/plasTeX/Base/LaTeX/Math.py#L124).
